mlir/lib/Dialect/Transform/Transforms/TransformInterpreterPassBase.cpp

```diff
@@ -379,14 +379,20 @@ static LogicalResult defineDeclaredSymbols(Block &block, ModuleOp definitions) {
 LogicalResult transform::detail::interpreterBaseRunOnOperationImpl(
     Operation *target, StringRef passName,
     const std::shared_ptr<OwningOpRef<ModuleOp>> &sharedTransformModule,
-    const std::shared_ptr<OwningOpRef<ModuleOp>> &libraryModule,
+    const std::shared_ptr<OwningOpRef<ModuleOp>> &transformLibraryModule,
```

Comment:
I could rename that as well, but I guess it would also make sense to rename the corresponding accessor (`getTransformLibraryModule`) and renaming *that* would make the PR non-NFC. Preferences?

Reply:
I might do this in a different PR.